test.sh

```diff
@@ -0,0 +1,120 @@
+#! /bin/bash
+
+## Colors for pretty printing in console output
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+NOCOLOR='\033[0m'
+
+## This function changes into the test_latex directory.
+## If this is not possible (for instance, if the directory does not exist) it fails with an error message and terminates the script.
+change_into_latex_dir () {
+	echo -e -n "${GREEN}change into latex dir...${NOCOLOR}"
+	if cd test_latex ; then
+		echo -e "${GREEN}OK${NOCOLOR}"
+	else
+		echo -e "${RED}FAIL${NOCOLOR}"
+		exit 1
+	fi
+}
+
+## This function deletes all files but .tex files in the current working directory.
```

Comment:
Add to comment that it is non-recursive? So only top-level tex files survive.

Reply:
done